src/journal/views.py

```diff
@@ -1016,6 +1017,7 @@ def publish_article(request, article_id):
     :param article_id: Article PK
     :return: contextualised django template
     """
+    from submission import forms as submission_forms
```

Comment:
Does having this import in the head cause a circular import?

Reply:
Yes I believe so. There is another function in this file that has the same import.